src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/IdeaCollection/Proposal/Create.html

```diff
@@ -32,7 +32,7 @@
         </label>
 
         <!-- Detail Description -->
-        <label data-ng-if="hasDetailText">
+        <label data-ng-if="processProperties.hasDescription">
```

Comment:
The attribute `hasDescription` needs to be added to the `processProperties` interface, of course. (The commit got lost.) This happens in d1b36e4.